### PR TITLE
Add support to lsvpd 1.7.7

### PR DESCRIPTION
**Pull request by olavphibm**
And dependency libvpd 2.2.5

### CODE REVIEW

**Comment by olavphibm** [comment]
test_plz

**Comment by olavphibm** [comment]
test_plz

**Comment by olavphibm** [comment]
test_plz

**Comment by olavphibm** [comment]
test_plz

**Comment by olavphibm** [comment]
test_plz

**Comment by olavphibm** [comment]
test_plz

File: lsvpd/lsvpd.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Package:
+ name: 'lsvpd'
+ clone_url: 'http://git.code.sf.net/p/linux-diag/lsvpd'
```
Comment (maurorodrigues) [inline lsvpd/lsvpd.yaml]: The idea here is to use https://github.com/open-power-host-os/lsvpd.git instead, unless rmm or lsvpd folks think different (which would mean, imho, remove lsvpd from the open-power-host-os namespace).
Other than that, pretty good stuff, thanks!

File: libvpd/libvpd.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Package:
+ name: 'libvpd'
+ clone_url: 'http://git.code.sf.net/p/linux-diag/libvpd'
```
Comment (maurorodrigues) [inline libvpd/libvpd.yaml]: as it is for lsvpd, the idea here is to use  https://github.com/open-power-host-os/libvpd.git instead, again, unless developers or rmm think different.

**Comment by olavphibm** [comment]
Thanks @maurorodrigues, fixed the URLs and testing again.

**Comment by olavphibm** [comment]
test_plz

**Pull request merged**